### ISSUE TITLE
Search Bar Tab Order Discrepancy

### ISSUE THREAD

**Issue by wesleyboar** [TACC/Core-CMS#1124]
**Describe the bug**
Keyboard focus can tab to search field, but [accessibility tool](https://www.ssa.gov/accessibility/andi/help/install.html) reports it not in tab order.

**To Reproduce**
1. [Install ANDI.](https://www.ssa.gov/accessibility/andi/help/install.html)
2. Open https://tacc.utexas.edu/.
3. Activate ANDI.
4. In ANDI, click "tab order ⏿".
5. Notice search field shown without a tab index.

**Expected behavior**
Search field should have a tab index.

**Screenshots**

<img width="900" height="475" alt="Image" src="https://github.com/user-attachments/assets/42a1e313-fbe8-41e5-a569-2c74ceca4ded" />

https://github.com/user-attachments/assets/b0beb400-cdad-457f-9136-ebf32c5b69cb

**Desktop (please complete the following information):**
 - Browser Chromium-based 145

**Additional context**
Reported by UT, who is also confused.

> [!TIP]
> I suspect search bar's shadow DOM is the culprit.
